Gems/Atom/Feature/Common/Assets/Materials/Pipelines/Common/DepthPass.shader.template

```diff
@@ -0,0 +1,20 @@
+{
+    "Source" : "INSERT_AZSL_HERE",
+
+    "DepthStencilState" : { 
+        "Depth" : { "Enable" : true, "CompareFunc" : "GreaterEqual" }
```

Comment:
Assuming reverse-z is probably a safe assumption but we can keep this in the back of our minds (in some cases, shader implementation will be coupled to PSO state like this)

Reply:
Also do we want >= and not > here?

Reply:
This is just a duplicate of the main depth shader (Gems\Atom\Feature\Common\Assets\Shaders\Depth\DepthPass.shader), so I won't change it here. That said, I don't remember the history behind deciding to use reverse-z or why we have >=